webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+	entry: './formatter/index.js',
+	output: {
+		path: path.resolve(__dirname, './backend/static/js/formatter/'),
+		filename: 'index.js',
+	},
+	mode: 'development',
```

Comment:
What are the differences between `development` and `production` (and are the other modes)? Which is most suitable for us here?

Reply:
As far as i can tell, the main difference is the names of modules and chunks.
There is also the `none` mode, which opts out of any optimizations.
I would default to `production`. I think useful module names are useful for stacktraces and so on, which we will probably not have anyway. But i could very well miss something here.

Reply:
The WASM itself is identical in all modes. This only affects the JS loader, so it doesn't make a big size difference.